doc/sec_paper/Intersections.tex

```diff
@@ -198,14 +198,22 @@
     & s_+ = k-s(\xt)+1-k \equiv 1-s_a \mod m, \\
     & s_- = k-s(\xt)-k \equiv -s_a \mod m.
     \end{align*}
-  The case $\varphi < 0$ is easily derived by symmetry: if we mirror Figure \ref{m-fig:int_a=c} at the horizontal line through $a$ we are in case (iv).
+    
+  Case (iv): \\
+  When $\varphi < 0$ we can use the antisymmetry of the intersection pairing to
+  fall back to case (iii) by looking at the negative of the intersection of the swapped cycles
```

Comment:
looking at the intersection of the swapped cycles (the minus sign is in the equation, that is enough)